test/Microsoft.AspNetCore.Routing.Tests/EndpointConstraints/EndpointConstraintEndpointSelectorTest.cs

```diff
@@ -506,5 +507,66 @@ public void OnProvidersExecuted(EndpointConstraintProviderContext context)
             {
             }
         }
+
+        private class HttpMethodEndpointConstraint : IEndpointConstraint
```

Comment:
copy-pasted this here to keep these tests running for now. I'll be moving this code to MVC soon.

Reply:
Yeah sorry, what I mean is that I plan on moving ALL of the endpoint constraint-related stuff out. So the code EndpointConstraintEndpointSelector and friends is moving back to MVC.